source/clear-linux/tutorials/fmv.rst

```diff
@@ -0,0 +1,268 @@
+.. _fmv:
+
+Use the Function Multi Version patch generator
+##############################################
+
+CPU architectures often gain interesting new instructions as they evolve but
+application developers find it difficult to take advantage of those
+instructions. The reluctance to lose backward-compatibility is one of the
+main roadblocks slowing developers from using advancements in newer computing
+architectures. :abbr:`FMV (Function multi-versioning)`, which first appeared
+in GCC 4.8, is a way to have multiple implementations of a function, each
+using a different architecture's specialized instruction-set extensions. GCC
+6 introduces changes to FMV to make it even easier to bring architecture-
+based optimizations to the application code.
+
+In this tutorial we will use FMV on general code and on :abbr:`FFT Fast
+Fourier Transform` library code. Upon completing the tutorial, you will be
+able to use this technology on your code and use the libraries to deploy
+architecture-based optimizations to your application code.
+
+Install and configure a Clear Linux host on bare metal
+******************************************************
+
+First, follow our guide to :ref:`bare-metal-install`.
+
+Once the bare metal installation and initial configuration are complete,
+add the `desktop-dev` bundle to the system.
+
+desktop-dev: contains the necessary development tools like GCC\* and Perl\*.
```

Comment:
Use literal text for the bundle name.
The \* is not necessary for GCC. Also note the asterisks should be implemented on the first instance.

Reply:
Done